src/components/Slider/Slider.tsx

```diff
@@ -141,6 +169,8 @@ const Slider: FC<PropsWithChildren<SliderProps>> = (props) => {
       </div>
     </sliderContext.Provider>
   );
-};
+});
+
+Slider.displayName = 'Slider';
```

Comment:
i don't think it's needed here

Reply:
we need this because we are using `forwardRef`